dev-lang/vala/vala-0.36.11.recipe

```diff
@@ -75,18 +98,24 @@ INSTALL()
 {
 	make install
 
-	# remove libtool library files
-	rm $libDir/lib*.la
+	rm -f $libDir/*.la
 
 	fixPkgconfig
 
+	#devel and common packages
+	packageEntries devel \
+		$developDir \
+		$manDir
```

Comment:
Well, after all, I'm not sure we should move the man to the devel package.
Here all man pages are in man1. And all `cmd:*` items are in the base package.
So I'd say that the man pages should also be in the base package.

That said, if we are 100% sure that all `cmd:` items may be put in the devel package, then it would be OK to move both `$manDir` and `$binDir`. But that might be risky, so the easiest solution is to keep the `$manDir` in the base package.

Reply:
I already thought it was strange, not having the manpages installed together with the cmd's ... :)